README.md

```diff
@@ -265,6 +236,87 @@ The static site that gets built is pushed to the `documentation` branch of this
+
+### Cutting a Major / Minor release
+#### A week before release day
+
+1. **Make sure the [milestone](https://github.com/RasaHQ/rasa/milestones) already exists and is scheduled for the 
+correct date**
+2. **Take a look at the issues & PRs that are in the milestone**: does it look about right for the release highlights 
+we are planning to ship? Does it look like anything is missing? Don't worry about being aware of every PR that should 
+be in, but it's useful to take a moment to evaluate what's assigned to the milestone.
+3. **Post a message on the engineering Slack channel**, letting the team know you'll be the one cutting the upcoming 
+release, as well as:
+    1. Providing the link to the appropriate milestone
+    2. Reminding everyone to go over their issues and PRs and please assign them to the milestone
+    3. Reminding everyone of the scheduled date for the release
+
+#### Release day! 🚀
+
+1. **At the start of the day, post a small message on slack announcing release day!**. Communicate you'll be handling 
+the release, and the time you're aiming to start releasing (again, no later than 4pm, as issues may arise and 
+cause delays)
+2. **Go over the milestone and evaluate the status of any PR merging that's happening. Follow up with people on their 
+bugs and fixes.** If the release introduces new bugs or regressions that can't be fixed in time, we should discuss on 
+Slack about this and take a decision to go forward or postpone the release. The PR / issue owners are responsible for 
+communicating any issues which might be release relevant.
+3. Once everything in the milestone is taken care of, post a small message on Slack communicating you are about to 
+start the release process (in case anything is missing).
+4. **You may now do the release by following the instructions outlined in the 
+[Rasa Open Source README](#steps-to-release-a-new-version) !**
+
+### Steps to release a new version
+Releasing a new version is quite simple, as the packages are build and distributed by GitHub Actions.
+
+*Terminology*:
+* patch release (third version part increases): 1.1.2 -> 1.1.3
+* minor release (second version part increases): 1.1.3 -> 1.2.0
+* major release (first version part increases): 1.2.0 -> 2.0.0
+
+*Release steps*:
+1. Make sure all dependencies are up to date (**especially Rasa SDK**)
+    - For Rasa SDK that means first creating a [new Rasa SDK release](https://github.com/RasaHQ/rasa-sdk#steps-to-release-a-new-version) (make sure the version numbers between the new Rasa and Rasa SDK releases match)
+    - Once the tag with the new Rasa SDK release is pushed and the package appears on [pypi](https://pypi.org/project/rasa-sdk/), the dependency in the rasa repository can be resolved (see below).
+2. Switch to the branch you want to cut the release from (`master` in case of a major / minor, the current feature branch for patch releases) 
```

Comment:
```suggestion
2. Switch to the branch you want to cut the release from (`master` in case of a major / minor, the current feature branch for micro releases) 
```